app/helpers/administrate/application_helper.rb

```diff
@@ -4,6 +4,16 @@ module ApplicationHelper
 
     def render_field(field, locals = {})
       locals.merge!(field: field)
+      if field.class.field_type == "has_one"
```

Comment:
There should be polymorphic way to avoid such conditions.
Since we already get `to_partial_path` from field itself, why not move generation of partial locals to field too?